app/routes/events/components/EventAdministrate/AttendeeElements.js

```diff
@@ -115,23 +114,21 @@ export const Unregister = ({
   fetching,
   handleUnregister,
   id,
-  clickedUnregister,
-  showUnregister
+  clickedUnregister
 }: UnregisterProps) => {
   return (
     <div>
-      {showUnregister &&
```

Comment:
Why is this check no longer needed?

Reply:
The column will now disappear instead of being empty.